book/src/configuration/notifications.md

```diff
@@ -38,4 +40,12 @@ Notification should trigger a OS toast.
 
 - **type**: boolean
 - **values**: `true`, `false`
-- **default**: `false`
\ No newline at end of file
+- **default**: `false`
+
+## `smart`
```

Comment:
i think we should call it delay instead, since smart sounds like... its smart :D

Reply:
Done! `delay` makes way more sense :)